code/modules/assembly/flash.dm

```diff
@@ -205,6 +205,10 @@
 	if(victim.loc == attacker.loc)
 		return DEVIATION_PARTIAL
 
+	// Ensures that when a moth is flashed, it will always be at no deviation. This is to make directionality irrelevant to moths.
+	if(ismoth(victim))
```

Comment:
It makes more sense to tie this to the fact that moth eyes are sensitive to light than it is to tie this to the moth species. That makes it affect more than just moths *generally* (something that'll need to be documented), but that's fine since it doesn't often come up anyway.

There are two ways of solving this. We can either tie this to a trait with moth eyes, or check if `get_eye_protection() < 0`. The former makes it, generally, work on moths always while the latter won't work if they have sunglasses on (something that usually has basically no impact for moths, who need to wear full welding gear). Let me know which you think is the better solution.

Furthermore--your comment isn't great, it's essentially rewriting the code but in different words.